### PR TITLE
Add the messages

### PR DESCRIPTION
**Pull request by germanruzca**
# Description
Add the errors messages to the correct inputs
Also show alerts and notice with styles from bootstrap

### CODE REVIEW

**Review by Javi1499 — APPROVED** [review]
LGTM

**Review by nallely64 — DISMISSED** [review]
LGTM!

File: app/assets/stylesheets/components/_post_form.scss [hunk 1]
```diff
@@ -55,26 +56,14 @@
       }
     }
 
-    label,
-    p {
-      font-family: "Roboto";
+    label, p {
+      font-family: 'Roboto';
```
Comment (juperameza) [inline app/assets/stylesheets/components/_post_form.scss:60]: Consider using a variable

**Review by nallely64 — DISMISSED** [review]
LGTM!

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/components/_post_form.scss (hunk 1 above)
Comment (germanruzca) [reply app/assets/stylesheets/components/_post_form.scss]: Done!, Thanks

**Review by juperameza — APPROVED** [review]
LGTM

**Review by Javi1499 — APPROVED** [review]
LGTM

**Pull request merged**